test/unit/app/models/admin/statistics_announcement_filter_test.rb

```diff
@@ -15,6 +15,14 @@ class Admin::StatisticsAnnouncementFilterTest < ActiveSupport::TestCase
                  filter.statistics_announcements
   end
 
+  test "ordering is scoped to a statistics announcements current_release_date" do
+    statistics_announcement1 = statistics_announcement_for(1.hour.from_now)
+    statistics_announcement2 = statistics_announcement_for(5.hours.from_now)
+    create(:statistics_announcement_date, release_date: 10.hours.from_now)
```

Comment:
Does this need to be applied to `statistics_announcement1`? 🤔 Otherwise I'm confused what this line is doing.

```suggestion
    create(:statistics_announcement_date, statistics_announcement: statistics_announcement1, release_date: 10.hours.from_now)
```

Reply:
Good spot. I don't know why but i thought default ordering was asc, but it's desc to i was getting a false positive. Have added this ☝️ and altered the ordering to reflect stat ann1 should be first.